Grammar/python.gram

```diff
@@ -144,20 +144,20 @@ import_from[stmt_ty]:
 import_from_targets[asdl_alias_seq*]:
     | '(' a=import_from_as_names [','] ')' { a }
     | import_from_as_names !','
-    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, CHECK(alias_ty, _PyPegen_alias_for_star(p))) }
+    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, _PyAST_alias(CHECK(PyObject *, _PyPegen_alias_star(p)), NULL, EXTRA)) }
```

Comment:
I think I'd prefer it if we pass `EXTRA` to `_PyPegen_alias_star` and let that construct the AST node. That's what we've been doing at other places, so let's be consistent with that.
```suggestion
    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, CHECK(alias_ty, _PyPegen_alias_for_star(p, EXTRA))) }
```

Reply:
Agreed 👍

Reply:
Fair enough. Done.